pkgs/os-specific/darwin/apple-source-releases/Libsystem/default.nix

```diff
@@ -56,9 +56,40 @@ appleDerivation rec {
     # The startup object files
     cp ${Csu}/lib/* $out/lib
 
-    # OMG impurity
```

Comment:
Didn't really feel like the comment added much value, since it's fairly obvious that it's impure. If we had a convention for specially marked comments so we can grep for impurity, that'd be nice, but I don't know how I'd enforce that.